cloudflare/resource_cloudflare_logpush_job.go

```diff
@@ -118,6 +119,12 @@ func resourceCloudflareLogpushJobCreate(d *schema.ResourceData, meta interface{}
 		return fmt.Errorf("error finding logpush job: %v", err)
 	}
 
+	var re = regexp.MustCompile(`^((datadog|splunk)://|s3://.+endpoint=)`)
```

Comment:
I wonder if this would be better suited in `getJobFromResource` as the CRUD methods all use it?

Reply:
Sounds good! I avoided this initially just incase there was a scenario where this could prevent you deleting an existing resource, given we're checking this on create though I think you're right - it may as well be moved. 😄